CharlieBackend.Core/DTO/Homework/CreateHometaskDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace CharlieBackend.Core.DTO.Homework
+{
+    public class CreateHometaskDto
+    {
+        public ushort? DeadlineDays { get; set; }
+
+        public string TaskText { get; set; }
+
+        public bool Common { get; set; }
+
+        [Required]
+        public long ThemeId { get; set; }
+
+        [Required]
+        public long MentorId { get; set; }
```

Comment:
only two fields Required? add other validation like string length etc.

Reply:
Yes, strictly required is only Theme an Mentor. But Rest lines are optional. String length of TaskText can be up to VARCHARMAX, so i left it empty